docs/release-notes/5.7.44-49.md

```diff
@@ -0,0 +1,118 @@
+# Percona Server for MySQL 5.7.44-49 (2024-02-XX)
+
+!!! important
+
+    Percona Server for MySQL 5.7.44-48 was the final release of the Percona Server for MySQL 5.7 series.
+
+    We recommend that you either [upgrade to MySQL 8.0 or stay on 5.7; we'll support you].
+
+
+## Release Information
+
+Percona Server for MySQL 5.7.44-49 contains the fix for [CVE-2024-20963] and a portability fix.
+
+The fixes are available to [Post MySQL 5.7 EOL support customers]. Community members can [build this release from source].
+
+## Bug fixes
+
+* [PS-9096]: Fixes the following:
+
+    * [Bug#35764496]: Running too much data through MD5() causes the server to stop. The cause was accessing invalid memory or over-allocating while computing hashes in the md5 encryption methods.
+
+    * [Bug#35967676]: Fixed a compilation failure in VS2022.
+
+* [PS-9048]: In the Debug build, assertions with OPTIMIZE table and fulltext indexes failed.
+
+## CVE and Third-party library information
+
+!!! admonition "important"
+
+    The binary tarballs are not shipped with the updated third-party libraries.
+
+
+The following Common Vulnerabilities and Exposures (CVE) have been identified in the third-party libraries:
+
+  * [CVE-2023-38545]
+
+  * [CVE-2023-5363]
+  
+  We recommend [upgrading the affected third-party library] to the latest version, at least curl v8.4.0 and OpenSSL v3.0.12. Also, the operating systems may port these fixes into the default versions of the libraries installed in the systems. For example, the patched version on the following platforms fix the CVE for OpenSSL:
+
+  * [Ubuntu 22.04 LTS]
+
+  * [Debian 12]
```

Comment:
[Oracle linux 9] can be added here with the url, https://linux.oracle.com/errata/ELSA-2024-0310.html.